Detections/SecurityEvent/UserPrincipalNameAssignedToUserAccount.yaml

```diff
@@ -0,0 +1,39 @@
+id: 875d0eb1-883a-4191-bd0e-dbfdeb95a464
+name: UserPrincipalNameAssignedToUserAccount
+description: |
+  'This query identifies whether a Active Directory user object was assigned a user principal name which could indicate that an adversary is preparing for performing Kerberoasting. 
```

Comment:
Should this be -  user object was assigned a service principal name instead of user principal name in the description.